endpoints/yomomma.py

```diff
@@ -0,0 +1,48 @@
+from flask import jsonify
+
+from utils.endpoint import Endpoint, setup
+from random import choice
+
+
+@setup
+class YoMomma(Endpoint):
+    """
+    This endpoint only returns a yo momma joke. No parameters are required.
+    """
+    params = []
+
+    def generate(self, avatars, text, usernames, kwargs):
+        choices = ['Yo momma so fat when she walked past the TV I missed three episodes',
```

Comment:
Perhaps these should be exported to a txt file split by new line to keep the endpoint looking clean(er)?

Reply:
Yeah might be better, will do that later